### DISCUSSION TITLE
Quyen Vu - Simple OpenMM system for testing code development

### DISCUSSION THREAD

**Discussion by giscus** [qvv5013/qvv5013.github.io#2] [General]
# Quyen Vu - Simple OpenMM system for testing code development

https://qvv5013.github.io/posts/openMM/openmm_code_snippet.html

**Comment by vuqv** [comment]
To parallelize the simulation across 2 GPUs, give it a comma-separated list of indices: `'DeviceIndex':'0,1'`.
